dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -609,5 +612,62 @@ public void VerifyUIElementsInItemsSource()
                 }
             });
         }
+    
+        [TestMethod]
+        public void VerifyRepeaterDoesNotLeakItemContainers()
+        {
+            ObservableCollection<int> items = new ObservableCollection<int>();
+            for(int i=0;i<10;i++)
+            {
+                items.Add(i);
+            }
+
+            ItemsRepeater repeater = null;
+
+            RunOnUIThread.Execute(() =>
+            {
+                var template = (DataTemplate)XamlReader.Load("<DataTemplate xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation' "
+                    + "xmlns:local='using:MUXControlsTestApp.Samples'>"
+                    + "<local:DisposableUserControl Number='{Binding}'/>" 
+                    + "</DataTemplate>");
+                Verify.IsNotNull(template);
+                Verify.AreEqual(0, MUXControlsTestApp.Samples.DisposableUserControl.OpenItems, "Verify we start with 0 DisposableUserControl");
+
+                repeater = new ItemsRepeater() {
+                    ItemsSource = items,
+                    ItemTemplate = template,
+                    VerticalAlignment = VerticalAlignment.Top,
+                    HorizontalAlignment = HorizontalAlignment.Left
+                };
+
+                Content = repeater;
+                
+            });
+
+            IdleSynchronizer.Wait();
+            // Wait until the Tick to make sure all layout has completed.
+            RunOnUIThread.WaitForTick();
+
+            RunOnUIThread.Execute(() =>
+            {
+
+                Verify.IsGreaterThanOrEqual(MUXControlsTestApp.Samples.DisposableUserControl.OpenItems, 10, "Verify we created at least 10 DisposableUserControl");
+
+                // Clear out the repeater and make sure everything gets cleaned up.
+                Content = null;
+                repeater = null;
+            });
+
+            IdleSynchronizer.Wait();
+            // Wait until the Tick to make sure all layout has completed.
+            RunOnUIThread.WaitForTick();
```

Comment:
I am not entirely sure if we need this, on my machine the test worked fine without it, but maybe this is depending on different factors, such as CPU usage etc.

Reply:
You might be right, i'll try removing it.